operator/controllers/btpoperator_controller_test.go

```diff
@@ -176,19 +176,6 @@ var _ = Describe("BTP Operator controller", Ordered, func() {
 					secret, err := createSecretWithoutKeys()
 					Expect(err).To(BeNil())
 					Eventually(k8sClient.Create(ctx, secret)).WithTimeout(k8sOpsTimeout).WithPolling(k8sOpsPollingInterval).Should(Succeed())
-					Eventually(getCurrentCrStatus).
```

Comment:
It is a mistake to expect two states (two assertions). If we expect "error" we should not expect "processing"